de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/NegotiationPacketListener.java

```diff
@@ -279,14 +279,14 @@ private void projectNegotiationCanceled(final JID sender,
         }
     }
 
-    private void projectNegotiationRequest(final JID sender,
+    private void referencePointNegotiationRequest(final JID sender,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this method into the anonymous class declared at line 129. ([squid:S3398](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3398))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJqzvkEeD8QRkTSt3&open=AWcmJqzvkEeD8QRkTSt3)

[ISSUE_KEY]: AWcmJqzvkEeD8QRkTSt3